pipelinedeploy.tex

```diff
@@ -63,42 +58,37 @@ \subsubsection{The ``Weekly Release'' pipeline}
 published, which is used to refer to a specific set of packages at fixed
 versions.  Examples of \texttt{eups} tags are: \texttt{w\_2018\_14}, \texttt{d\_2018\_04\_10}.
 
-The \texttt{eupspkg} are pushed to an AWS S3 ``bucket'', which is used as the
+\noindent The \texttt{eupspkg} are pushed to an AWS S3 ``bucket'', which is used as the
 canonical repository for both source and binary \texttt{eups} packages.  The S3
 bucket is not publicly accessible as \texttt{eups} only supports remote
 package repositories over \texttt{http[s]} that have HTML ``index
 pages'' in the traditional format produced by the \texttt{apache}
 webserver.  To accommodate this
 requirement, the S3 bucket is periodically replicated to a simple
-service that
-deploys on kubernetes\footnote{\url{https://kubernetes.io/}} cluster which consists
-of \texttt{TLS} termination, a reverse proxied Apache instance, a persistent
-storage volume, and a replication script.  The publicly accessible URL for
-source packages is presently \url{https://eups.lsst.codes/stack/src/}.
-
-\paragraph{Tag git repos.}
+service that deploys on a Kubernetes cluster providing \url{https://eups.lsst.codes/stack/src/}.
 
+\noindent{\bf Tag git repos.}
 A script contained in the
-\href{https://github.com/lsst-sqre/sqre-codekit}{\texttt{sqre-codekit}} Python package
+\href{https://github.com/lsst-sqre/sqre-codekit}{\texttt{sqre-codekit}} package
 uses the initial build ``manifest'' to apply an annotated git tag to all
 repositories from which an \texttt{eups} product was pulled as part of the
 build along with a number of ancillary git repositories that contain build
 scripts, documentation, and other items that would be needed to exactly reproduce the
 release.
+%%WOM above could be shortened
 
-\paragraph{Build/publish eups binary ``tarball'' packages.}
-
+\noindent{\bf Build/publish eups binary ``tarball'' packages.}
 This is perhaps the most complex stage in a release.  Multiple builds/publishes
 are run in parallel.  As of the \texttt{15.0} (\texttt{eups} distrib tag
 \texttt{v15\_0}) release, the build matrix is: Python 2 and 3 on CentOS~6, CentOS~7, and macOS 10.9.
 
-Every build is started from scratch and bootstraps a development/installation environment using the \href{https://github.com/lsst/lsst/blob/master/scripts/newinstall.sh}{\texttt{new\-install.sh}}\cite{pipelines-guide} script, just as an ``end user'' would when building/installing science pipeline software. Eg.
+\noindent Every build is started from scratch and bootstraps a development/installation environment using the \href{https://github.com/lsst/lsst/blob/master/scripts/newinstall.sh}{\texttt{new\-install.sh}}\cite{pipelines-guide} script, just as an ``end user'' would when building/installing science pipeline software e.g.
 
 \begin{verbatim}
```

Comment:
This curl line is definitely on the chopping block.

Reply:
yes looks ugly - I have removed it